design/ceph-managed-disruptionbudgets.md

```diff
@@ -0,0 +1,44 @@
+# Handling node drains through managed PodDisruptionBudgets.
+
+## Goals
+
+- Handle and block node drains that would cause data unavailablity and loss.
+- Unblock drains dynamically so that a rolling upgrade is made possible.
+- Allow for rolling upgrade of nodes in automated kubernetes environments like [cluster-api](https://github.com/kubernetes-sigs/cluster-api)
+
+
+
+## Design
+
```

Comment:
Can you add more detail as to why PDB and maxUnavailable in MachineConfigPool does not work for Ceph OCS during OCP upgrades?

Reply:
I'll add more details.